roles/mac80211hwsim/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Kernel Module
+- name: Add the mac80211_hwsim module
+  community.general.modprobe:
+    name: mac80211_hwsim
+    state: absent
```

Comment:
Is is necessary to unload module before loading it ?

If yes, it would be nice to add a short comment to explain reason.

Reply:
The module could have been installed and loaded with a different option.
By disable it first, the second part will be effective. modprob with another option is not changing anything with https://docs.ansible.com/ansible/latest/collections/community/general/modprobe_module.html

Reply:
It makes sense